apps/right-to-rent-check/views/step.html

```diff
@@ -0,0 +1,8 @@
+{{<partials-page}}
+  {{$page-content}}
+    {{#fields}}
+      {{#renderField}}{{/renderField}}
+    {{/fields}}
+    {{#input-submit}}continue{{/input-submit}}
+  {{/page-content}}
+{{/partials-page}}
```

Comment:
Does bootstrap still throw an error if you don't include any views? We should really fix this if so, this file isn't needed